docs/feature_secure.md

```diff
@@ -26,12 +26,12 @@ SECURE_ENABLE = yes
 
 ## Keycodes
 
-| Key              | Description                                                                    |
-|------------------|--------------------------------------------------------------------------------|
-| `SECURE_LOCK`    | Revert back to a locked state                                                  |
-| `SECURE_UNLOCK`  | Forces unlock without performing a unlock sequence                             |
-| `SECURE_TOGGLE`  | Toggle directly between locked and unlock without performing a unlock sequence |
-| `SECURE_REQUEST` | Request that user perform the unlock sequence                                  |
+| Key                 |Aliases  | Description                                                                    |
+|---------------------|---------|--------------------------------------------------------------------------------|
+| `QK_SECURE_LOCK`    |`SE_LOCK`| Revert back to a locked state                                                  |
+| `QK_SECURE_UNLOCK`  |`SE_ULCK`| Forces unlock without performing a unlock sequence                             |
+| `QK_SECURE_TOGGLE`  |`SE_TOGG`| Toggle directly between locked and unlock without performing a unlock sequence |
+| `QK_SECURE_REQUEST` |`SE_REQ` | Request that user perform the unlock sequence                                  |
```

Comment:
`SE_RQST` maybe?
```suggestion
| `QK_SECURE_REQUEST` |`SE_RQST`| Request that user perform the unlock sequence                                  |
```

Reply:
`REQ` is fine, doesn't always have to be 4 chars.